packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1273,6 +1273,12 @@ export default class Meeting extends StatelessWebexPlugin {
   setUpLocusInfoMeetingListener() {
     this.locusInfo.on(EVENTS.REMOTE_ANSWERED_DECLINED, (payload) => {
       this.meetingFiniteStateMachine.remote(payload);
+
```

Comment:
Please add an info that we are attempting to leave the meeting here:

```suggestion
LoggerProxy.logger.info('Meeting:index#setUpLocusInfoMeetingListener --> REMOTE_ANSWERED_DECLINED. Attempting to leave meeting.');
```